packages/jest/src/default-config.resolver.spec.ts

```diff
@@ -11,6 +11,13 @@ describe('Resolve project default configuration', () => {
         expect(config.globals['ts-jest'].tsConfig).toEqual(getSystemPath(normalize(`/some/cool/directory/${tsConfigName}`)));
     });
 
+    it('Should resolve path to the tsconfig if "tsConfig" is provided', () => {
+        const defaultConfigResolver = new DefaultConfigResolver('./projects/my-cool-library/tsconfig.spec.json');
```

Comment:
It is pretty confusing, took me a while to understand what's going on here. Maybe it would be better if path to the tsConfig relative to the project root didn't contain `projects/my-cool-library`. Probably something like `./ts-configs/my-tsconfig.json` would make more sense.

Reply:
Sure, no problem.